tests/examples/source/devfiles/nodejs/devfile-with-endpoints.yaml

```diff
@@ -0,0 +1,70 @@
+commands:
```

Comment:
Have you considered using [`devfile-with-multiple-endpoints.yaml`](https://github.com/redhat-developer/odo/blob/main/tests/examples/source/devfiles/nodejs/devfile-with-multiple-endpoints.yaml)?

Reply:
Nope - I looked for similar Devfiles, but didn't see this file. Looks like it has been introduced recently, in 0396b43.
But that's a good point. Now that it's there, I will use it instead.